api-specificatie/openapi.yaml

```diff
@@ -894,17 +905,17 @@ components:
 
     ZoekResultaat:
       type: object
-      description: 'Dit type bevat een enkel zoekresultaat bestaande uit een weergave tekst en zoekresultaat identificatie'
+      description: 'Dit type bevat een enkel zoekresultaat bestaande uit een omschrijving en zoekresultaat identificatie'
       properties:
-        weergavenaam:
-          description: 'Weergavenaam van het zoekresultaat'
-          title: 'weergavenaam'
+        omschrijving:
+          description: 'Omschrijving van het zoekresultaat'
+          title: 'omschrijving'
```

Comment:
waarom heb je de title opgenomen? Die is identiek aan de naam van de parameter.

Reply:
Die title heb ik niet opgenomen, die stond er al. Ik zal de spec even doorlopen op overbodige titles.

Reply:
Ik stel voor daar dat ik daar een issue voor inschiet. De hele spec staat vol met overbodige titles (Die dus gelijk zijn aan de property-naam).